packages/verticaltabs/src/react/group.tsx

```diff
@@ -57,11 +48,11 @@ interface GroupStatics {
 interface GroupComponent
   extends RefForwardingComponent<GroupProps, HTMLLIElement, GroupStatics> {}
 
-const Group = React.forwardRef((props, ref) => {
+const Group = React.forwardRef(({ className, ...props }, ref) => {
```

Comment:
We destructure `className` here? Seems to disallow styling of Group and nothing else.

Reply:
Error on my part